src/elements/content-sidebar/MetadataSidebarRedesign.scss

```diff
@@ -1,5 +1,8 @@
 @import '../common/variables';
+@import '../../styles/variables';
 
 .bcs-MetadataSidebarRedesign {
+    width: 340px;
```

Comment:
shouldn't it take the width of a parent container?

Reply:
Probably, but now if we don't set the width, the sidebar is smaller than it should be.